frontend/src/ui/OnetLink.stories.tsx

```diff
@@ -0,0 +1,18 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+import OnetLink, { OnetLinkProps } from './OnetLink';
+
+export default {
+  title: 'JobHopper/OnetLink',
+  component: OnetLink,
+} as Meta;
+
+export const OnetLink_15dash1131point00: Story<OnetLinkProps> = args => (
```

Comment:
Please use camelCase here and name it after the occupation (computer programmer) rather than the code to improve readability.

Reply:
updating in commit.